arelle/CntlrWinMain.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, parent):
         self.labelLang = overrideLang if overrideLang else self.modelManager.defaultLang
         self.data = {}
 
+        addInstalledPluginModules()
```

Comment:
Does this always force enable pip installed plugins? That would limit the usability of this mechanism. For instance, we wouldn't be able to add ixbrl-viewer to our requirements build file and drop all of the node build steps from the Arelle workflows.

It would be better if this instead made pip installed plugins show up in the "Select Plug-in Module" dialog for enabling and disabling